.travis.yml

```diff
@@ -10,14 +10,11 @@ addons:
     - libssl-dev
     - libunwind8
     - zlib1g
-before_install:
-  - if test "$TRAVIS_OS_NAME" == "osx"; then brew update; brew install icu4c; fi
```

Comment:
cc @victorhurdugaci - would you know why this works without installing icu? The template doesn't have this and other repos have been succeeding.